website/blog/2020-06-27-lithium.md

```diff
@@ -0,0 +1,527 @@
+
+- Import other scripts: Metals provides completions to select existing files
+  based on the filesystem information
+
+```scala
+//main.sc
+import $file.hello-world
+val greeting = hello-world.greeting
+
+//hello-world.sc
+val greeting = "Hello world!"
+```
+
+- Import dependecies:
+
+```scala
+import $ivy.`com.lihaoyi::scalatags:0.7.0`
+import scalatags.Text.all._
+
+val rendered = div("Moo").render
+```
+
+## Worksheet dependency support
+
+In addition to Ammonite support, which will be useful to anyone interested in
+Scala scripting, we've also improved the support for Metals Scala worksheets.
+Thanks to the efforts of [olafurpg](https://github.com/olafurpg) in
+[mdoc](https://scalameta.org/mdoc/), which powers worksheets, users are now able
+to import dependencies directly inside of their Scala worksheet with completions
+and diagnostics working right from the start.
+
+![worksheet-imports](https://i.imgur.com/qXgdNWM.gif)
+
+You can add them using the following syntax:
+
+```
+$dep.`organisation::artifact:version`
+```
+
+For example:
+
+```
+$dep.`com.lihaoyi::scalatags:0.9.0`
+```
+
+`::` is the same as `%%` in sbt, which will append the current Scala binary
+version to the artifact name.
+
+## New project provider
+
+Creating new Scala projects is usually a bit of work to get started, which is a
+problem for both beginners and experts alike. Normally you don't want to spend a
+lot of time figuring out the best setup for starting with a new technology or
+the language itself.
+
+There are great tools like [giter8](http://www.foundweekends.org/giter8/), which
+is used in the `sbt new` command, that enables people to use already existing
+Scala templates found on Github such as
+[https://github.com/scala/scala-seed.g8](https://github.com/scala/scala-seed.g8).
+The problem is when you don't use sbt or you don't know the exact template you
+want to use.
+
+To address this, we've added an additional command called `New Scala Project`
+that enables users to reuse the same mechanism from the comfort of your editor.
+Firstly, you can choose from existing templates, either curated by the Metals
+team, ones existing on the giter8
+[wiki page](https://github.com/foundweekends/giter8/wiki/giter8-templates) or
+alternatively an entirely custom templates. Then you can choose the exact
+location for the new project and the name. Afterwards, Metals will run giter8 to
+create the project and offer to open that new project.
+
+![new-project](https://user-images.githubusercontent.com/3807253/81842954-2d1b3680-954d-11ea-934a-96fe53cafeb9.gif)
+
+The command is also available as a button in the welcome view of the Visual
+Studio Code Metals extension, which means it will be visible in case there are
+no imported projects in the current workspace.
+
+The first curated templates were choosen by the team, however we are happy to
+include more so that it's easier for people to discover new amazing libraries or
+frameworks.
+
+## Multiline string improvements
```

Comment:
Added another section that might be useful for people to see. Especially the code action, which might not be obvious, but is super useful. I thinking of adding a short one about the multiple build tools maybe later.

Reply:
@ckipp01 or do you think the short mention is enough for that feature?

Reply:
I think this is a really good idea having the subsection. I think I'd even suggest adding a small gif and short description of the "choose build tool one". I'm of of the mindset that anything that changes the general UX of something should be mentioned, and since that one does, it might be good to add it. Then we could also mention things like:
- What scenario is this good for? Ex. When you have a project with both a build.sbt and build.sc, but the build.sc is what you use for your build definition. Previously if both were detected, it was automatically considered an sbt workspace. Now you get to choose.
- If you choose the wrong one, or change later one, you'll need to remove the `.metals/` directory to reset the decisions.

Reply:
Added the next section, I think this is mostly ready. Will go over it once more once we are ready to release.